### ISSUE TITLE
Inconsistent A11y state description after adding isotope on Single Atom screen

### ISSUE THREAD

**Issue by vperezcadavid99** [phetsims/alpha-decay#130]
**Test device**
Asus Zenbook

**Operating System**
Win 11

**Browser**
Firefox 

**Problem description**
From https://github.com/phetsims/qa/issues/1434

During the Single Atom screen, after adding an isotope, the state description reads: 'Currently Isotope A atom in decay area. Add atom to begin observations.'

Having 'Add atom to begin observations' present while the live context response simultaneously announces 'Isotope A added' seems inconsistent.

**Steps to reproduce**
1. Go the A11y view
2. Go to the first screen.
3. Change to Custom Isotope.
4. Set the Half-life slider to a high value to increase decay duration, then pause the simulation.
5. See the state description "Add atom to begin observations." 

**Visuals**

<img width="2748" height="1096" alt="Image" src="https://github.com/user-attachments/assets/446caa7d-f946-486f-b755-9004921520d3" />

<details>
Troubleshooting information:
!!!!! DO NOT EDIT !!!!!
Name: ‪Alpha Decay‬
URL: https://phet-dev.colorado.edu/html/alpha-decay/1.0.0-dev.8/phet/alpha-decay_all_phet.html
Version: 1.0.0-dev.8 2026-08-28 14:22:50 UTC
Features missing: applicationcache, applicationcache, touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/152.0.0.0 Safari/537.36
Language: en-US
Window: 1600x812
Pixel Ratio: 1.7999999523162842/1
WebGL: WebGL 1.0 (OpenGL ES 2.0 Chromium)
GLSL: WebGL GLSL ES 1.0 (OpenGL ES GLSL ES 1.0 Chromium)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 30 uniform: 4096
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 32767x32767
OES_texture_float: true
Dependencies JSON: {}
<details/>

**Comment by Nancy-Salpepi** [comment]
Thanks @vperezcadavid99.
We currently have 2 interaction hints--one to get things started and one to restart things after decay has occurred, so they can continue to explore. 

There are other sims that have static interaction hints that are no longer accurate after the user starts interacting. A couple of examples:
"Start Sunlight" is always the interaction hint for Greenhouse Effect.
"Turn on light source" is always the interaction hint for Molecules and Light.
"Place skater on track to begin exploring." for Energy Skate Park.

So I think it is OK as is. 
If all sounds good to you, please close.

**Comment by vperezcadavid99** [comment]
@Nancy-Salpepi thanks for the context.
Closing.
